fast_llm/config.py

```diff
@@ -916,3 +925,69 @@ def __init__(self, config: ConfigType, *args, **kwargs):
     @property
     def config(self) -> ConfigType:
         return self._config
+
+
+@config_class()
+class TypeableConfig(Config):
```

Comment:
Probably best left for later (#126), especially since it's not strictly needed for the present PR.